### PR TITLE
[NET-0031] Implement codiga as code quality checker

### LINKED ISSUE #33: NET-0031: Implement automated code checking
**DoD**
- [x] Automated linting
- [x] Automated code quality check

**Relevant Story**
As a developer, I want a clean repository, to reduce project size, and for better code readability/documentability.

### PR DESCRIPTION
**Pull request by Abelbait**
## Description
- Implement codiga as code quality checker
- Sample workflow: https://github.com/allendomingo/StoreInventory/actions/runs/1743606944
- Quality conditions:
-       min_quality_grade: 'WARNING'
        min_quality_score: '50' (out of 100)
        max_defects_rate: '0.05' (5 out of 100 lines)
        max_complex_functions_rate: '0.05' (5 out of 100 functions)
        max_long_functions_rate: '0.05' (5 out of 100 functions)

### CODE REVIEW

**Review by RayAdrian — DISMISSED** [review]
Quality 💯 👍

File: .github/workflows/Codiga.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Codiga
+on:
+  push:
+    branches: [ NET-0031-codiga ]
```
Comment (RayAdrian) [inline .github/workflows/Codiga.yml]: Shouldn't this be all branches?

**Review by RayAdrian — APPROVED** [review]
Quality 💯 👍

File: .github/workflows/Codiga.yml (hunk 1 above)
Comment (Abelbait) [reply .github/workflows/Codiga.yml]: Changed to pull requests only for dev branch. Please check again :)

**Pull request merged**